aospy/internal_names.py

```diff
@@ -0,0 +1,56 @@
+"""Internal names used throughout aospy"""
```

Comment:
Addressed in a677b93

I ended up being a little more careful about the distinction between variables and coordinates here; let me know what you think.